src/profile/index.unit.tsx

```diff
@@ -10,62 +10,75 @@ import Text, { TextDisplayType } from 'text'
 const defaultProps = {
   title: 'Jack Sparrow',
 }
+describe('Profile', () => {
+  let defaultProfile
 
-it('Should pass a title prop to Item', () => {
-  const profile = shallow(<Profile title="Jack Sparrow" />)
-  expect(profile.find(Item).prop('leftTitle')).toEqual('Jack Sparrow')
-})
+  beforeEach(() => {
+    defaultProfile = shallow(<Profile {...defaultProps} />)
```

Comment:
Used in too few test cases to be in `beforeEach`. In most of your tests, you'll end up shallowing twice the component

Reply:
I splitted tests in two sections.